lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -229,6 +229,7 @@ date.autoformatter.hour   : %H:%M:%S
 date.autoformatter.minute : %H:%M:%S.%f
 date.autoformatter.second : %H:%M:%S.%f
 date.autoformatter.microsecond : %H:%M:%S.%f
+date.epoch                : 0000-12-31T00:00:00  # old epoch
```

Comment:
I would not add `date.epoch` to styles because I see it as a functional property not a style property.

Btw. what would happen if I change the style later after having already used date operations?

Reply:
Styles are just rcParams.  The tests are run using the classic style.  This is how you tell all the tests to run with the old epoch.  The alternative is changing every date test which I'm fine with doing, but there will be a lot of changes.

Reply:
https://github.com/matplotlib/matplotlib/blob/f091a4d92e0a73e702e15c172c884d35c9a478a2/lib/matplotlib/testing/conftest.py#L33

That is where we do the setup / clean up if we want to force this in the tests.

I think I agree that this is not something that should be in the style sheets (and probably should be added to the banned list).

Reply:
OK, blacklisted and changed the tests to use the new epoch.